docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  autoretrieve:
+    build: .
+    environment:
+      FULLNODE_API_INFO: ${FULLNODE_API_INFO:-wss://api.chain.love}
+      AUTORETRIEVE_PER_MINER_RETRIEVAL_LIMIT: $AUTORETRIEVE_PER_MINER_RETRIEVAL_LIMIT
+      AUTORETRIEVE_ENDPOINT: $AUTORETRIEVE_ENDPOINT
+      AUTORETRIEVE_ENDPOINT_TYPE: $AUTORETRIEVE_ENDPOINT_TYPE
```

Comment:
Upon adding the `AUTORETRIEVE_ENDPOINT_TYPE` env var to my command, I now get a different error about the mount volume not being absolute.
```
$ DOCKER_BUILDKIT=1 AUTORETRIEVE_ENDPOINT_TYPE=indexer docker-compose up
...
Error response from daemon: invalid mount config for type "volume": invalid mount path: 'data' mount path must be absolute
```

Not sure what's going on here, because it looks like the mount volume was fine on the first run since it actually ran autoretrieve but failed on a bad config option. 🤔